CactbotOverlay/FateWatcherKo.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Machina.FFXIV;
+using Machina.FFXIV.Headers;
+
+namespace Cactbot {
+  class FateWatcherKo : FateWatcher {
+    //
+    // for FFXIV KO version: 5.01
+    //
+    // Latest KO version can be found at:
+    // https://www.ff14.co.kr/news/notice?category=3
+    //
+    private const ushort Server_MessageTypeActorControl143 = 323;
+
+    public FateWatcherKo(CactbotEventSource client) : base(client) {
```

Comment:
I think for things like memory structrures, it makes sense to have different ffxivprocess classes, but I don't think the pattern really makes sense here, especially as it's just two values? I think this would be better if either:
* FateWatcher had constructor parameters for these values and they got passed in
* FateWatcher had a constructor value for the language, and it just set the values internally from the language

...and then we don't need multiple classes hanging around that duplicate a bunch of logic.

Reply:
I've consolidated this into a single FateWatcher class with help from @ngld's OverlayPlugin code.